openlibrary/templates/account/reading_log.html

```diff
@@ -1,4 +1,4 @@
-$def with (docs, key, shelf_count, doc_count, owners_page, current_page, sort_order='desc', user=None, include_ratings=False, q=None, results_per_page=25, ratings=[])
+$def with (docs, key, shelf_count, doc_count, owners_page, current_page, sort_order='desc', user=None, include_ratings=False, q=None, results_per_page=25, ratings=[], checkin_year=None)
```

Comment:
If `checkin_year` we can likely hide the pager -- I'm assuming we don't limit to 25 in this case, we probably want to confirm.

Reply:
We were limiting, but I removed the restriction for yearly summaries.